docs/source/doctest_fixtures.py

```diff
@@ -115,6 +115,7 @@ def _make_synthetic_bert_state():
     pin_memory=False,
     drop_last=False,
 )
+eval_dataloader2 = eval_dataloader
```

Comment:
Curious does this need to be a global doctest fixture? Or could it be part of a `.. testsetup::`?

Reply:
Yea let's definitely move this to a `.. testsetup:: ` directive since it's just making a copy of the existing `eval_dataloader`.